common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -0,0 +1,33 @@
+﻿@maa_01 = "gfx/interface/icons/culture_innovations/innovation_maa_01.dds"
```

Comment:
The camel icon is out of style.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105386210-8df72480-5c2d-11eb-8f15-0241fa8662e5.png)

</details>

Reply:
The elekks too.